### PR TITLE
Fix databricks doc

### PR DESCRIPTION
**Pull request by bharanidharan14**
Addressed the review comments given in this PR https://github.com/astronomer/astronomer-providers/pull/718

### CODE REVIEW

**Comment by bharanidharan14** [comment]
@phanikumv Corrected the doc and created the PR

File: astronomer/providers/databricks/operators/databricks.py [hunk 1]
```diff
@@ -17,9 +17,9 @@ class DatabricksSubmitRunOperatorAsync(DatabricksSubmitRunOperator):
     Submits a Spark job run to Databricks using the
     `api/2.1/jobs/runs/submit
     <https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsRunsSubmit>`_
-    API endpoint. Using DatabricksHook It makes two non-async API calls to
-    submit the run, and retrieve the run page URL. By getting the job id from the response polls for the status
-    execute the Databricks trigger, and defer execution as expected.
+    API endpoint. Using DatabricksHook it makes two non-async API calls to
+    submit the run, and retrieve the run page URL, by getting the job id from the response polls for the status
```
Comment (phanikumv) [inline astronomer/providers/databricks/operators/databricks.py:21]:
```suggestion
    submit the run, and retrieve the run page URL. By getting the job id from the response, polls for the status
```

**Comment by rajaths010494** [comment]
hey @bharanidharan14  if this issue is resolved. Can u close the ticket along with tagging the PR?

**Comment by bharanidharan14** [comment]
@rajaths010494 I have addressed the review comment, need approval on this PR

**Review by rajaths010494 — APPROVED** [review]
LGTM.

**Comment by bharanidharan14** [comment]
@phanikumv can you review this PR

File: astronomer/providers/databricks/operators/databricks.py [hunk 2]
```diff
@@ -17,9 +17,9 @@ class DatabricksSubmitRunOperatorAsync(DatabricksSubmitRunOperator):
     Submits a Spark job run to Databricks using the
     `api/2.1/jobs/runs/submit
     <https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsRunsSubmit>`_
-    API endpoint. Using DatabricksHook It makes two non-async API calls to
-    submit the run, and retrieve the run page URL. By getting the job id from the response polls for the status
-    execute the Databricks trigger, and defer execution as expected.
+    API endpoint. Using DatabricksHook it makes two non-async API calls to
```
Comment (phanikumv) [inline astronomer/providers/databricks/operators/databricks.py:20]:
```suggestion
    API endpoint. Using DatabricksHook, it makes two non-async API calls to
```

**Pull request merged**